numba/tests/test_withlifting.py

```diff
@@ -279,6 +279,9 @@ def test_liftcall5(self):
         # capture all stdout
         with captured_stdout() as stream:
             njit(liftcall5)()
+        # A single 'A' character will be printed
+        self.assertIn('A', str(stream.getvalue()))
```

Comment:
For consistency perhaps use `check_same_semantic` as elsewhere (and model the test on those above which are similar)?

Reply:
Right, this `assertIn` pattern was originally for lifting case that would fail.